sites/kit.svelte.dev/package.json

```diff
@@ -3,7 +3,7 @@
 	"private": true,
 	"version": "0.0.1",
 	"scripts": {
-		"update": "node scripts/check-doc-links.js && node scripts/types",
```

Comment:
why remove the call to `scripts/check-doc-links.js`?

Reply:
It does not handle the IPFS required relative links. AFAIK, kit prerenderer already checks all the links, so we should be fine. I removed this in svelte.dev a long time back and never found any issues with it

Reply:
We should probably delete the script as being unused now then?